src/EditorFeatures/CSharpTest/Diagnostics/HideBase/HideBaseTests.cs

```diff
@@ -34,7 +34,7 @@ class App : Application
 
 class App : Application
 {
-    public static new App Current { get; set; }
+    public new static App Current { get; set; }
```

Comment:
Interesting.  I'm ok with this change in behavior for now.  But this does indicate an area where SyntaxGenerator does need to be better.  But it's so minor we don't need to fix in this PR.